infrastructure/src/main/java/org/corfudb/infrastructure/BatchWriter.java

```diff
@@ -259,17 +306,28 @@ private void batchWriteProcessor() {
                                 streamLog.reset();
                                 res.add(currOp);
                                 break;
+                            case LOG_TAIL_QUERY:
+                                Long logTail = streamLog.getLogTail();
```

Comment:
Is it possible that `getLogTail()` throws an exception? What happens if it does?

Reply:
There is a catch around this that will treat any exception.